ass/__main__.py

```diff
@@ -0,0 +1,130 @@
+"""Blend a subtitle onto a png image
+"""
+
+import argparse
+import os
+import sys
+
+import numpy as np
+from PIL import Image
```

Comment:
Does this library really need to have a hard dependency on pil and numpy?

Reply:
Yes, they are necessary to run the test. `numpy` handles the array containing the bitmap image, while `Pillow` writes the `PNG` image.

Reply:
Unit tests shouldn't require the users of the library to install these dependencies

Reply:
This test is taken from the original `libass` repo, so we can consider it an integration test. Perhaps we could move that in the `tests` directory.